Project Euler/Q5 Project Euler

```diff
@@ -0,0 +1,24 @@
+#include<stdio.h>
+int main()
+{
+        int n;
+        scanf("%d",&n);
+        for(int i=n;;i++)
+        {
+                int flag=1;
+                for(int j=1;j<=n;j++)
+                {
+                //      int flag=1;
+                        if(i%j!=0)
+                        {
+                                flag=0;
+                                break;
+                        }
+                }
+                if(flag==1)
```

Comment:
Just a style thing but comparison to 1 is something many people don't want to see.

`if (flag)` is equivalent. (You're asking "is it true, that this is true?" instead of "is this true?")